src/__internal__/focus-trap/focus-trap.component.js

```diff
@@ -38,16 +38,20 @@ const FocusTrap = ({
   );
 
   useLayoutEffect(() => {
-    const elements = Array.from(
-      ref.current.querySelectorAll(defaultFocusableSelectors)
-    ).filter((el) => Number(el.tabIndex) !== -1);
-
-    if (hasNewInputs(elements)) {
-      setFocusableElements(Array.from(elements));
-      setFirstElement(elements[0]);
-      setLastElement(elements[elements.length - 1]);
+    if (wrapperRef) {
+      const ref = wrapperRef.current || wrapperRef;
+
+      const elements = Array.from(
+        ref.querySelectorAll(defaultFocusableSelectors)
```

Comment:
I don't think it's going to work if `wrapperRef` is a function (as described in propTypes)

Reply:
yep copy and pasted from above 🤦